src/Features/Core/Portable/SignatureHelp/SignatureHelpOptions.cs

```diff
@@ -3,18 +3,12 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.CodeAnalysis.Completion;
-using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.SignatureHelp
 {
     internal readonly record struct SignatureHelpOptions(
         bool HideAdvancedMembers)
     {
-        public static SignatureHelpOptions From(Project project)
-            => From(project.Solution.Options, project.Language);
-
-        public static SignatureHelpOptions From(OptionSet options, string language)
-          => new(
-              HideAdvancedMembers: options.GetOption(CompletionOptions.Metadata.HideAdvancedMembers, language));
+        public static readonly SignatureHelpOptions Default = new(CompletionOptions.Default.HideAdvancedMembers);
```

Comment:
interesting that we have one completion option that affects sighelp.  on well... presumable how it's worked for ages.

Reply:
Yeah, HideAdvancedMembers affects multiple things. Perhaps it might make sense to separate it from completion options.

Reply:
https://github.com/dotnet/roslyn/issues/59197